e2e/tests/stepDefinitions/lostBeds.ts

```diff
@@ -4,23 +4,35 @@ import { LostBedCreatePage } from '../../../cypress_shared/pages/manage'
 import lostBedFactory from '../../../server/testutils/factories/lostBed'
 import referenceDataFactory from '../../../server/testutils/factories/referenceData'
 
+Given('I navigate to the lost beds create page', () => {
+  cy.get('.moj-button-menu__toggle-button')
+    .click()
+    .then(() => cy.get('a').contains('Record a lost bed').click())
+})
+
 Given('I create a lost bed', () => {
   const lostBed = lostBedFactory.build({
     startDate: new Date(2022, 1, 11, 0, 0).toISOString(),
     endDate: new Date(2022, 2, 11, 0, 0).toISOString(),
     reason: referenceDataFactory.build({ id: Cypress.env('lost_bed_reason_id') }),
   })
 
-  cy.get('.moj-button-menu__toggle-button')
-    .click()
-    .then(() => cy.get('a').contains('Record a lost bed').click())
-
-  const lostBedCreatePage = new LostBedCreatePage(lostBed)
+  const lostBedCreatePage = new LostBedCreatePage()
 
   lostBedCreatePage.completeForm(lostBed)
   lostBedCreatePage.clickSubmit()
 })
 
+Given('I attempt to create a lost bed without the necessary information', () => {
+  const lostBedCreatePage = new LostBedCreatePage()
+  lostBedCreatePage.clickSubmit()
+})
+
 Then('I should see a notification that the lost bed has been created', () => {
   cy.get('.govuk-notification-banner').should('contain', 'Lost bed logged')
 })
+
+Then('I should see a list of the problems encountered creating the lost bed', () => {
+  const lostBedCreatePage = new LostBedCreatePage()
+  lostBedCreatePage.shouldShowErrorMessagesForFields(['startDate', 'endDate', 'numberOfBeds', 'reason'])
```

Comment:
I don't think the matcher

```
shouldShowErrorMessagesForFields(fields: Array<string>): void {
    fields.forEach(field => {
      cy.get('.govuk-error-summary').should('contain', errorLookups[field]?.empty)
      cy.get(`[data-cy-error-${field}]`).should('contain', errorLookups[field]?.empty)
    })
  }
```

works to verify that none of the "reason" radios have been selected. Cypress is telling me:

```
AssertionError
Timed out retrying after 4000ms: 
Expected to find element: input[name="lostBed[reason]"][value="undefined"], 
but never found it.
```

Reply:
Ah, sorry, that's down to some missing env vars